app/pages/cif/project/[project]/index.tsx

```diff
@@ -4,24 +4,140 @@ import { graphql, usePreloadedQuery } from "react-relay/hooks";
 import { ProjectOverwiewQuery } from "__generated__/ProjectOverwiewQuery.graphql";
 import withRelayOptions from "lib/relay/withRelayOptions";
 
-const pageQuery = graphql`
+export const pageQuery = graphql`
```

Comment:
I was going to change the variable name to `projectOverviewQuery`, but then it occurred to me that a name change might cause problems elsewhere?